polaris-react/.storybook/main.js

```diff
@@ -32,6 +32,7 @@ module.exports = {
           {
             loader: 'babel-loader',
             options: {
+              configFile: path.resolve(__dirname, '../../babel.config.js'),
```

Comment:
```suggestion
              configFile: path.join(__dirname, '../../babel.config.js'),
```

**suggestion (non-blocking):** `__dirname` is already an absolute path so no need to `.resolve()` one

Reply:
Good catch, ty!